var/spack/repos/builtin/packages/musica/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Musica(CMakePackage):
+    """MUSICA - The multi-scale interface for chemistry and aerosols
+
+    MUSICA is a software package designed which exposes a flexible
+    API for including aerosol and gas-phase chemistry in
+    many contexts across languages and platforms. It is designed to
+    be used in conjunction with other software packages, such as
+    climate models, to provide a comprehensive framework for
+    simulating atmospheric processes.
+    """
+
+    homepage = "https://github.com/NCAR/musica"
+    git = "https://github.com/NCAR/musica"
+
+    maintainers("kshores", "mattldawson", "boulderdaze")
+
+    license("Apache-2.0", checked_by="kshores")
+
+    # Versions
+    version("0.10.1", tag="v0.10.1")
```

Comment:
Is there a reason for using a tag (instead of a hash)?  The tag will not result in the preferred trusted download (see https://spack.readthedocs.io/en/latest/packaging_guide.html#trusted-downloads).

Reply:
You could add a `sha256` if you add the `url` property around line 20 (see https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching).

Reply:
For example, if you add the URL for the included version (your GitHub repository -> releases -> version -> assets -> Source Code (.tar.gz)), you can run `spack checksum musica` to get the sha256, check checksums (option `c`), and get the complete `version` directive.

Reply:
Oh, there was no reason other than my ignorance of what I was doing. Thank you for the suggestion! I have updated this to use a checksum instead.